Libs/Optimize/ParticleSystem/itkParticleImageDomain.h

```diff
@@ -148,6 +148,7 @@ class ParticleImageDomain : public ParticleRegionDomain
   }
 
   inline PointType GetValidLocationNear(PointType p) const override {
+    // todo why is this function ignoring the argument? Also see Optimize::AddSinglePoint
```

Comment:
It seems to be only used to place the first point with:
```

  PointType firstPointPosition;
  firstPointPosition[0] = 0;
  firstPointPosition[1] = 0;
  firstPointPosition[2] = 0;
  firstPointPosition = m_sampler->GetParticleSystem()->GetDomain(0)->GetValidLocationNear(firstPointPosition);
```

We should probably rename these functions based on their current usage.